collections/ansible_collections/vmware/nsx_alb/plugins/modules/avi_vcenterserver.py

```diff
@@ -0,0 +1,148 @@
+description:
+    - This module is used to configure VCenterServer object
+    - more examples at U(https://github.com/avinetworks/devops)
+requirements: [ avisdk ]
+version_added: "2.7"
+options:
+    state:
+        description:
+            - The state that should be applied on the entity.
+        default: present
+        choices: ["absent", "present"]
+        type: str
+    avi_api_update_method:
+        description:
+            - Default method for object update is HTTP PUT.
+            - Setting to patch will override that behavior to use HTTP PATCH.
+        default: put
+        choices: ["put", "patch"]
+        type: str
+    avi_api_patch_op:
+        description:
+            - Patch operation to use when using avi_api_update_method as patch.
+        choices: ["add", "replace", "delete"]
+        type: str
+    cloud_ref:
+        description:
+            - Vcenter belongs to cloud.
+            - It is a reference to an object of type cloud.
+            - Field introduced in 20.1.1.
+        type: str
+    content_lib:
+        description:
+            - Vcenter template to create service engine.
+            - Field introduced in 20.1.1.
+        required: true
+        type: dict
+    name:
+        description:
+            - Availabilty zone where vcenter list belongs to.
+            - Field introduced in 20.1.1.
+        required: true
+        type: str
+    tenant_ref:
+        description:
+            - Vcenter belongs to tenant.
+            - It is a reference to an object of type tenant.
+            - Field introduced in 20.1.1.
+        type: str
+    url:
+        description:
+            - Avi controller URL of the object.
+        type: str
+    uuid:
+        description:
+            - Vcenter config uuid.
+            - Field introduced in 20.1.1.
+        type: str
+    vcenter_credentials_ref:
+        description:
+            - Credentials to access vcenter.
+            - It is a reference to an object of type cloudconnectoruser.
+            - Field introduced in 20.1.1.
+        required: true
+        type: str
+    vcenter_url:
+        description:
+            - Vcenter hostname or ip address.
+            - Field introduced in 20.1.1.
+        required: true
+        type: str
+extends_documentation_fragment:
+    - vmware.nsx_alb.avi
+'''
+
+EXAMPLES = """
+- name: Example to create VCenterServer object
+  vmware.nsx_alb.avi_vcenterserver:
+    controller: 192.168.15.18
+    username: admin
+    password: something
+    state: present
+    name: sample_vcenterserver
+"""
+
+RETURN = '''
+obj:
+    description: VCenterServer (api/vcenterserver) object
+    returned: success, changed
+    type: dict
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+try:
+    from ansible_collections.vmware.nsx_alb.plugins.module_utils.avi import (
+        avi_common_argument_spec, avi_ansible_api, HAS_AVI)
+except ImportError:
```

Comment:
@amolopcito  I guess we could get rid of this check now that sdk code is already inbuilt.